### ISSUE TITLE
Question about Model

### ISSUE THREAD

**Issue by ChengkaiYang** [vita-epfl/UniTraj#30]
Dear @Alan-LanFeng 
        Glad to communication this excellent work with you again!
        In Unitraj, you provided three classical framework Autobot, MTR and WayFormer. I've reproduced Autobot's results on AV2 dataset, which got almost same brier-FDE(2.5) to result in your paper. However, on AV2 leaderboard, it didn't show competitive among models. After taken serious consideration, I've found all models' loss function in Unitraj were calculated on only one ego agent but not among all agents in scenario. In fact, some previous SOTA methods on AV2 single agent motion forecasting leaderboard like QCNet utilizing all agents in loss function but not only the ego one. Do you think this limited accuracy on single agent's motion forecasting or not?

**Comment by ChengkaiYang** [comment]
How about using 'interested agents' into training in AV2 dataset? Could this increase accuracy?

**Comment by Alan-LanFeng** [comment]
Hi,

We are using the 'focal agent' in AV2 to train the model.

UniTraj only reports vehicles' bfde in the paper, this might cause the performance discrepancy

**Comment by Alan-LanFeng** [comment]
interested agent == FOCAL_TRACK
